composer.json

```diff
@@ -15,15 +16,29 @@
 		"squizlabs/php_codesniffer": "^3.6",
 		"wp-coding-standards/wpcs": "^2.3",
 		"yoast/phpunit-polyfills": "^1.0",
-		"thor-juhasz/phpunit-coverage-check": "^0.3.0"
+		"thor-juhasz/phpunit-coverage-check": "^0.3.0",
+		"humbug/php-scoper": "0.17.5"
```

Comment:
Is there more popular package, seems like this one is not stable `0.17` for production + without a lot of stars.
what about the license of the package?

Reply:
Let me look into this